tests/unit/mainwindow/test_prompt.py

```diff
@@ -48,9 +48,8 @@ def _get_prompt_func(path):
         return _get_prompt_func
 
     @pytest.mark.parametrize('steps, where, subfolder', [
-        (1, 'next', '..'),
-        (1, 'prev', 'c'),
```

Comment:
You can leave this one as it still should work. Going to the previous item once still falls in `c`.

Reply:
Are you saying line 52 still works or line 51.

Reply:
Line 52. Going `1` step in the `prev` direction should still leave us in the `c` folder.